docs/sdk/v3/guides/liquidity/02-modifying-position.md

```diff
@@ -0,0 +1,115 @@
+
+We then pass the Position instance as input to the `NonfungiblePositionManager`'s `addCallParameters` function:
+
+```js reference title="Getting the transaction calldata and parameters" referenceLinkText="View on Github" customStyling
+https://github.com/Uniswap/examples/blob/d6300e2db41f6a2c3e9c69860347c17c484232ba/v3-sdk/modifying-position/src/example/Example.tsx#L156-L163
+```
+
+Note how the function also requires an options object as its second parameter off type [`AddLiquidityOptions`](https://github.com/Uniswap/v3-sdk/blob/08a7c050cba00377843497030f502c05982b1c43/src/nonfungiblePositionManager.ts#L77). This is either of type [`MintOptions`](https://github.com/Uniswap/v3-sdk/blob/08a7c050cba00377843497030f502c05982b1c43/src/nonfungiblePositionManager.ts#L74) for minting a new position or [`IncreaseOptions`](https://github.com/Uniswap/v3-sdk/blob/08a7c050cba00377843497030f502c05982b1c43/src/nonfungiblePositionManager.ts#L75) for adding liquidity to an existing position. In the minting case, we want to populate `MintOptions`.
+
+The function returns the calldata as well as the value required to execute the transaction, which we then execute. The effect of the transaction is to mint a new Position NFT, which should then be visible on the list of position ids.
+
+### Adding liquidity to our position by using the `NonfungiblePositionManager`'s `addCallParameters` to get the data for making the add liquidity transaction, and then executing the transaction
+
+All of the liquidity adding logic can be found in the [`addLiquidity`](https://github.com/Uniswap/examples/blob/d6300e2db41f6a2c3e9c69860347c17c484232ba/v3-sdk/modifying-position/src/example/Example.tsx#L128) function. The function is very similar to `mintPosition`, except for some details that we will go over. Notice how the **Add Liquidity** button is disabled until a position is minted.
+
+The first difference to point out is that we do not need to give approval to the `NonfungiblePositionManager` to transfer our tokens as we have already done that when minting our position. 
+
+To start, we create the position by which we want to increase our current position:
+
+```js reference title="Creating the Position" referenceLinkText="View on Github" customStyling
+https://github.com/Uniswap/examples/blob/d6300e2db41f6a2c3e9c69860347c17c484232ba/v3-sdk/modifying-position/src/example/Example.tsx#L186-L188
+```
+
+Note how we pass `percentageToAdd` to add as a parameter, which creates a new position with a percentage of the amount of each token that we want to increase by. We do not create a new position, but instead use the new Position instance to consider how much liquidity we want to add to our current position.
```

Comment:
The wording here is a bit confusing here ad we title the code "Creating a Position" and then say "We do not create a new position". Maybe break this down into steps, explaining what `getPosition` does as well as why we want its result?

Also, the percentage isn't clear I think on what the units are = e.g. is 100% `1` or `100`? Is the increase based on 1.X or 0.X? I think we may want to modify the code here to be a bit cleaner, like using integers and having the increase be the amount stated, e.g. `25` would mean a 25% increase. Should also leave a comment in the config definition and explain here using an example amount IMO

Reply:
This function also increases based on the inputs - should it do it based on the current position instead? e.g. If I hit "increase" twice, I'll only see it go up once right?